.github/workflows/benchmarks.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           - libxslt_xpath
           - mbedtls_fuzz_dtlsclient
           - mbedtls_fuzz_dtlsclient_7c6b0e
-          - mruby_mruby_fuzzer
+          - mruby_mruby_fuzzer_8c8bbd
```

Comment:
@DonggeLiu Maybe something worth noting: the CI will not fail if the benchmark doesn't exist!

Reply:
Wow, I did not know this, thanks!
I reckon this is low priority because it does not affect checking fuzzers or running experiments.
WDYT?

Reply:
I reckon it's low priority as well, but maybe something to keep in mind since it might inadvertently mark a fuzzer as "working" for a non-existent benchmark.

Reply:
Yep, created this: https://github.com/google/fuzzbench/issues/2037